data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApiTest.java

```diff
@@ -41,12 +42,15 @@
       mock(CombinedChainDataClient.class);
   private final JavalinServer server = mock(JavalinServer.class);
   private final Javalin app = mock(Javalin.class);
-  private static final Integer THE_PORT = 12345;
+  private static final Integer THE_PORT = 5051;
 
   @BeforeEach
   public void setup() {
+    ArtemisConfiguration config =
+        ArtemisConfiguration.fromString(
+            "beaconrestapi.portNumber=5051\nbeaconrestapi.enableSwagger=false");
```

Comment:
can 5051 not be hardcoded in here

Reply:
it's just configuring the configuration object for the test suite... alternative?